rest/admin_api.go

```diff
@@ -452,7 +452,15 @@ func (h *handler) putRole() error {
 
 func (h *handler) deleteUser() error {
 	h.assertAdminOnly()
-	user, err := h.db.Authenticator().GetUser(mux.Vars(h.rq)["name"])
+	username := mux.Vars(h.rq)["name"]
```

Comment:
This will panic if the map doesn't have the `name` property.  I think this should do the `val, ok := mux.Vars(h.rq)["name"]` style, and if `!ok`, then return an error rather than panicking.  (btw I realize that the existing code had the same issue, but per boy-scout rule it's probably good to make it safer while in this code)

Reply:
Reading from a map with a non existent key will return the zero value of the value type (empty string in this case)

https://play.golang.org/p/K0qQWKaF_c

Reply:
Ok thanks for the clarification!